grafana/scylla-cql.2019.1.template.json

```diff
@@ -374,6 +374,159 @@
+            },
+            {
+                "class": "row",
+                "height": "25px",
+                "gridPos": {"h": 2},
+                "panels": [
+                    {
+                        "class": "text_panel",
+                        "content": "<h1 style=\"color:#5881c2; border-bottom: 3px solid #5881c2;\">Cross DC Information</h1>",
+                        "style": {}
+                    }
+                ],
+                "title": "New row"
+            },
+            {
+                "class": "row",
+                "panels": [
+                    {
+                        "class": "gauge_errors_panel",
+                        "description": "Using consistency level ONE in a query when there is more than one DC will hurt performance, queries may end in the non-local DC. Use Local-One instead",
+                        "targets": [
+                            {
+                                "expr": "floor(100 *sum(delta(scylla_query_processor_queries{instance=~\"[[node]]\",cluster=~\"$cluster|$^\", dc=~\"$dc\", shard=~\"[[shard]]\", consistency_level=\"ONE\"}[30s]))/sum(delta(scylla_query_processor_queries{instance=~\"[[node]]\",cluster=~\"$cluster|$^\", dc=~\"$dc\", shard=~\"[[shard]]\"}[30s]))) OR vector(0)",
+                                "format": "time_series",
+                                "hide": false,
+                                "instant": false,
+                                "interval": "",
+                                "intervalFactor": 1,
+                                "legendFormat": "aaa",
+                                "refId": "A"
+                            }
+                        ],
+                        "title": "CQL ONE Queries"
+                    },
+                    {
+                        "class": "ops_panel",
+                        "description": "Using consistency level ONE in a query when there is more than one DC will hurt performance, queries may end in the non-local DC. Use Local-One instead",
+                        "span": 3,
+                        "targets": [
+                            {
+                                "expr": "sum(delta(scylla_query_processor_queries{instance=~\"[[node]]\",cluster=~\"$cluster|$^\", dc=~\"$dc\", shard=~\"[[shard]]\",  consistency_level=\"ONE\"}[1m])) by ([[by]])",
+                                "format": "time_series",
+                                "hide": false,
+                                "intervalFactor": 2,
+                                "refId": "A"
+                            }
+                        ],
+                        "title": "CQL ONE CL Queries"
+                    },
+                    {
+                        "class": "gauge_errors_panel",
+                        "description": "Using consistency level QUORUM in a query when there is more than one DC will hurt performance, queries may end in the non-local DC. Use Local-QUORUM instead",
```

Comment:
"Using consistency level QUORUM in a query when there is more than one DC will hurt performance, queries may end in the non-local DC. Use Local-QUORUM instead"
-->
"Using consistency level QUORUM in a query when there is more than one DC may hurt performance, queries may end up in the non-local DC. Use LOCAL_QUORUM instead"